docs/UserGuide.md

```diff
@@ -67,13 +67,90 @@ FApro seeks to improve the quality of life of financial advisors (FAs). It allow
+![help format](images/helpFormat.png)
+
+#### Precise expected outcome on success:
+* Successful message: Opened help window
+* A pop-up window lists all the main commands and displays a link to FAPro's user guide
 
 ![help window](images/helpWindow.png)
 
+### Viewing risk assessment questionnaire: `questionnaire`
+
+Displays questions that are going to be used for generating client risk profile levels.
+
+#### Format: `questionnaire`
+
+![questionnaire format](images/questionnaireFormat.png)
+
+#### Precise expected outcome on success:
+* Successful message: Opened questionnaire window
+* A pop-up window shows the risk assessment multiple choice questions, including the grading and risk profile categories criteria
+
+![questionnaire window](images/questionnaireWindow.png)
+
+### Adds risk profile level to a contact:`riskprofile`
+
+Generates the client risk profile level based on their response of risk assessment questionnaire provided in Questionnaire tab and adds it to their contact.
+
+#### Format:
+* `riskprofile INDEX res/RESULT`
+
+#### Example commands:
+* `riskprofile 3 res/a,b,c,d,e,e,b,c`
+
+![riskprofile format](images/riskProfileFormat.png)
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+This command can also be used to update the client risk profile
+</div>
+
+#### Acceptable values for each parameter:
+* INDEX: Only accept positive integers less than the size of the contacts displayed
+* RESULT: Valid result format, 8 comma-separated characters from 'a' - 'e' (e,b,a,c,b,b,a,e)
+
+#### Precise expected outcome on success:
+* Successful message: "Added risk profile to Person: X", where X are the details of the person edited
+* A risk profile label with a specified color will be added to the contact.
+  There are 5 categories:
+  <span style="background-color:#4CAF50; color:white;">**Low**</span>,
+  <span style="background-color:#2196F3; color:white;">**Moderately Low**</span>.
+  <span style="background-color:#FFC107; color:white;">**Moderate**</span>,
+  <span style="background-color:#FF6600; color:white;">**Moderately High**</span>,
+  <span style="background-color:#F44336; color:white;">**High**</span>
+
+![riskprofile result](images/riskProfileResult.png)
+
+<div markdown="span" class="alert alert-warning">
+:warning: **Warning:**
+Once the risk profile label is added to the contact, it cannot be removed
+</div>
+
+#### Precise expected outputs on failure:
+If the RESULT is empty
+* Error message:
+```
+Invalid command format! 
+riskprofile: Adds the risk profile of the person identified by the index number used in the last person listing. 
+Existing risk profile will be overwritten by the input.
+Parameters: INDEX (must be a positive integer) res/[RESULT]
+Example: riskprofile 1 res/a,e,b,d,c,a,d,e
+```
+![invalid risk profile 1](images/invalidRiskProfile1.png)
+
+If the RESULT is not separated by commas or not in the range of 'a' - 'e'
+* Error message:
```

Comment:
Same thing about the formatting of error message! Small thing!

Reply:
Solved!
Thank you.